cmd/analyzer/main.go

```diff
@@ -83,6 +86,18 @@ func vpcConfigsFromFile(fileName string, inArgs *InArgs) (map[string]*vpcmodel.V
 	return vpcConfigs, nil
 }
 
+func translateCDtoConnectionSet(inArgs *InArgs) *common.ConnectionSet {
+	connection := common.NewConnectionSet(false)
+	if common.ProtocolStr(*inArgs.QProtocol) == common.ProtocolICMP {
+		connection.AddICMPConnection(*inArgs.QSrcMinPort, *inArgs.QSrcMaxPort, *inArgs.QDstMinPort, *inArgs.QDstMaxPort)
```

Comment:
why are src/dst ports sent to `AddICMPConnection` as icmp type and code? 
initially for simplicity, we do not have input icmp type / code, so should add ALL ICMP connections (use the defualt min and max type/code values from the `common` pkg).